src/main/java/seedu/address/logic/commands/FindPersonCommand.java

```diff
@@ -11,19 +12,36 @@
  */
 public class FindPersonCommand extends FindCommand<Person> {
 
+    public static final String MESSAGE_USAGE = COMMAND_WORD
+            + " person : Finds all persons matching the following parameters.\n"
+            + "To specify name, use the parameter n/[NAME] where [NAME] is a list of names, non-case-sensitive,"
+            + "separated by spaces.\n"
+            + "Parameters: n/[NAME]\n"
```

Comment:
Do you think we can change this so that it sticks to the parameter formatting we have? (i.e. "[" + PREFIX_NAME + "NAME]"